index.html

```diff
@@ -1057,10 +1069,21 @@ <h2><dfn>Predicted events</dfn></h2>
             <p>The number of events in the list and how far they are from the current timestamp are determined by
             the user agent and the prediction algorithm it uses.</p>
                 
-            <p>The events in the predicted event list will have increasing
-            <a href="https://dom.spec.whatwg.org/#dom-event-timestamp"><code>timeStamps</code></a>([[!WHATWG-DOM]]),
-            so the first event will have the smallest <code>timeStamp</code> which should still be greater than
-            the last event in the <a>coalesced event list</a>.</p>
+            <p>Predicted events MUST:</p>
+            <ul>
+                <li>have the same <code>pointerId</code>, <code>pointerType</code>, <code>isPrimary</code>,
+                    <a href="https://dom.spec.whatwg.org/#dom-event-istrusted"><code>isTrusted</code></a>,
+                    and <a href="https://dom.spec.whatwg.org/#event-target"><code>target</code></a> as
+                    the dispatched "parent" event</li>
+                <li>have <code>cancelable</code> and <code>bubbles</code> set to <code>false</code>
+                    (as these events will never be dispatched in isolation)</li>
```

Comment:
Not sure (as these events will never be dispatched in isolation) is particularly useful